plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1864,17 +1864,11 @@ elementgui.fluid.special_information=<html>Special information about the bucket:
 elementgui.fluid.event_flow_condition=Additional flowing condition
 elementgui.fluid.event_before_replacing_block=Before replacing a block
 elementgui.food.event_on_eaten=When food eaten
```

Comment:
We don't have food mod element here anymore, rename those properties from `elementgui.food.???` to `elementgui.item.food_???`

Reply:
Note 3: I didn't change the ID of label texts because it would give additional and useless work to translators.

Reply:
I'd say keeping the _food_ namespace is a dis-informative thing since you removed the corresponding mod element type definition.

Reply:
@KlemenDEV should we keep `food` or change them for `item`.